lib/matplotlib/tests/test_figure.py

```diff
@@ -170,7 +170,7 @@ def test_gca():
         # Changing the projection will throw a warning
         assert fig.gca(polar=True) is not ax3
     assert fig.gca(polar=True) is not ax2
-    assert fig.gca().get_geometry() == (1, 1, 1)
+    assert fig.gca().get_subplotspec().get_geometry() == (1, 1, 0, 0)
```

Comment:
should you check that the subplot spec is 1, 1, 1?

Reply:
I think that's an equivalent check?

Reply:
Ooops, I see. You are indeed correct...